x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/metric_item.tsx

```diff
@@ -7,30 +7,29 @@
 import React from 'react';
 import { Chart, Metric, MetricDatum } from '@elastic/charts';
 import { EuiLoadingContent, EuiPanel } from '@elastic/eui';
-import { FETCH_STATUS, isPending } from '../../../../../hooks/use_fetcher';
 
 export function MetricItem({
   data,
   id,
-  status,
-  hasData,
+  isLoading,
+  height = '124px',
 }: {
   data: MetricDatum[];
   id: number;
-  status: FETCH_STATUS;
-  hasData: boolean;
+  isLoading: boolean;
+  height?: string;
 }) {
   return (
     <div
       style={{
         resize: 'none',
         padding: '0px',
         overflow: 'auto',
-        height: '120px',
+        height,
         borderRadius: '6px',
       }}
     >
-      {!hasData && isPending(status) ? (
+      {data && isLoading ? (
```

Comment:
`data` will always be true as you pass `<MetricItem id={key} data={[metric]} isLoading={loadingStats} /> ` [here](https://github.com/elastic/kibana/pull/149101/files#diff-8e2c738e19199614fe5fdc76e6f63e53eaacf2c305f1c69cfcca63a10f51d867R163)

is this what you wanted?

Reply:
You're right, I'll check if data is empty in MetricItem